crates/iroha/tests/integration/extra_functional/multiple_blocks_created.rs

```diff
@@ -1,28 +1,42 @@
-use std::thread;
-
 use eyre::Result;
 use iroha::{
-    client::{self, Client},
-    data_model::{parameter::BlockParameter, prelude::*},
+    client::{self},
+    data_model::prelude::*,
 };
-use iroha_config::parameters::actual::Root as Config;
+use iroha_data_model::parameter::BlockParameter;
 use iroha_test_network::*;
 use iroha_test_samples::gen_account_in;
 use nonzero_ext::nonzero;
+use tokio::task::spawn_blocking;
 
-const N_BLOCKS: usize = 510;
+#[tokio::test]
+async fn multiple_blocks_created() -> Result<()> {
+    const N_BLOCKS: u64 = 500;
 
-#[ignore = "Takes a lot of time."]
-#[test]
-fn long_multiple_blocks_created() -> Result<()> {
     // Given
-    let (_rt, network, client) = Network::start_test_with_runtime(4, Some(10_965));
-    wait_for_genesis_committed(&network.clients(), 0);
-    let pipeline_time = Config::pipeline_time();
+    let network = NetworkBuilder::new()
+        .with_peers(4)
+        .append_instruction(SetParameter(Parameter::Block(
+            BlockParameter::MaxTransactions(nonzero!(1u64)),
+        )))
+        .start()
+        .await?;
+    let peer_one = network.peer();
+    let client_one = peer_one.alice_client();
+    let last_peer = network.peers().last().unwrap();
+    assert_ne!(peer_one.id(), last_peer.id());
 
-    client.submit_blocking(SetParameter::new(Parameter::Block(
-        BlockParameter::MaxTransactions(nonzero!(1_u64)),
-    )))?;
+    let mut events = last_peer.events();
+    tokio::spawn(async move {
+        while let Ok(e) = events.recv().await {
+            match e {
+                PeerLifecycleEvent::LogBlockCommitted { height } => {
+                    println!("Last peer block committed: {height}")
+                }
+                _ => {}
+            }
+        }
+    });
```

Comment:
Yes, this is only for debugging. It sometimes fails due to timeout being too short (which is convenient for local development). Need to adjust further.

Reply:
> Also `logger` is no longer available in integration tests, right?

`test_logger()` setup function is still available and will work as before, but I want to discourage its usage. Mainly because it requires initialisation before logs could be seen, which adds a layer of complexity to already complex tests.

If some advanced logging is needed (more advanced than `println!`), we can use `log` crate, for example. `tracing` is a bit too heavy imo.